beets/dbcore/query.py

```diff
@@ -841,8 +841,8 @@ def is_slow(self):
 
 class NullSort(Sort):
     """No sorting. Leave results unsorted."""
-    def sort(items):
-        return items
+    def sort(self):
+        return self
```

Comment:
This one should be:

```py
def sort(self, items):
    return items
```

That is, it's just missing an unused `self` parameter; it's not misnamed.